### DISCUSSION TITLE
How do I use `consult-line` (with live previews) with `ivy`?

### DISCUSSION THREAD

**Discussion by NightMachinery** [minad/consult#945] [Q&A] [answered]
How do I use `consult-line` (with live previews) with `ivy`?

Alternatively, if that's not supported, how do I use `consult-line` with, e.g., `vertico`, but only for this specific command?

I have most of my config using `ivy` and `counsel`, so I can't switch my defaults. I have no problems with simultaneously using both `ivy` and another completion framework.

PS: I know of `swiper`, `counsel-grep` etc. I don' know of a `counsel-rg` variant for the current file though; its default version searches the current dir.

**Comment by minad** [comment, marked as answer]
See my comments in #212. The situation hasn't changed since then.

EDIT: See @NightMachinery's solution at https://github.com/minad/consult/issues/212#issuecomment-1960316840.
